docs/sources/fundamentals/timeseries-dimensions/index.md

```diff
@@ -104,8 +104,6 @@ In the case of SQL-like data sources, more than one numeric column can be select
 Additional technical information on tabular time series formats and how dimensions are extracted can be found in [the developer documentation on data frames as time series](/developers/plugin-tools/introduction/data-frames#data-frames-as-time-series).
 
 {{% docs/reference %}}
```

Comment:
This doesn't seem related to plugin docs? Commenting just in case

Reply:
Awesome catch! This does indeed look wrong.